docs/guide/multisite.md

```diff
@@ -2,109 +2,202 @@
 
 ## Introduction
 
-With eZ Platform it is possible to serve multiple, completely different sites using one eZ Platform instance and database.
+You can serve multiple different sites using one eZ Platform instance and database.
 
-Each site will have its own content root, at a lower level than the default one (Location ID 2). One backoffice can be used for each site, but it is also possible to use a global one.
+Each site will have its own content root, at a lower level than the default one (Location ID 2).
+You can use one global back office for all sites, or a separate back office for each site.
 
-This feature is a reimplementation of the [PathPrefix](http://doc.ez.no/eZ-Publish/Technical-manual/4.x/Reference/Configuration-files/site.ini/SiteAccessSettings/PathPrefix) one that existed in eZ Publish Legacy.
+This feature is a reimplementation of the [PathPrefix](http://doc.ez.no/eZ-Publish/Technical-manual/4.x/Reference/Configuration-files/site.ini/SiteAccessSettings/PathPrefix) that existed in eZ Publish Legacy.
 
-## Configuration
+## Configuring multisite
 
-The feature is configured in `ezplatform.yml`, either at siteaccess or siteaccess group level:
+Multisite is configured in `ezplatform.yml`, either at [SiteAccess](siteaccess.md) or SiteAccess group level:
 
 ``` yaml
 ezpublish:
     system:
-        ezdemo_site_group:
+        site_group:
             content:
                 tree_root:
                     # Root locationId. Default is top locationId
                     location_id: 123
-                    # Every URL aliases starting with those prefixes will be considered
+                    # All URL aliases starting with those prefixes will be considered
                     # being outside of the subtree starting at root_location.
                     # Default value is an empty array.
                     # Prefixes are not case sensitive.
                     excluded_uri_prefixes: [ /media, /images ]
 ```
 
-##### `content.tree_root.location_id`
+### Configuration parameters
 
-This parameter sets the Location ID of the content root. The API is jailed within this root, and nothing above this level will be accessible by default. This parameter can be filtered using the `excluded_uri_prefixes` parameter described below.
+The two configuration parameters, `location_id` and `excluded_uri_prefixes` are taken into account in several places,
+for example in URL alias lookups for incoming web requests, and URL alias path generation in Twig templates.
+In addition you need to consider them when generating site paths in your layout, or other places rendering site/tree structure.
 
-##### `content.tree_root.excluded_uri_prefixes`
+#### `location_id`
```

Comment:
`location_id` is treated on the previous paragraph. What about changing the order of these paragraphs?

Reply:
The previous paragraph is an overview of the two available parameters, while this and the next section cover the parameters in detail.